driftbase/tests/test_parties.py

```diff
@@ -292,6 +292,34 @@ def test_leaving_party_invalidates_outstanding_invites_from_same_player(self):
         g2_notification, g2_message_number = self.get_party_notification('invite')
         self.patch(g2_notification['invite_url'], data={'inviter_id': host_id}, expected_status_code=http_client.NOT_FOUND)
 
+    def test_invite_to_full_party_is_not_allowed(self):
+        # Create players for test
+        guest_ids = []
+        guest_names = []
+
+        for guest_name in [self.make_user_name(f"Guest_{i}") for i in range(4)]:
+            self.make_named_player(username=guest_name)
+            guest_ids.append(self.player_id)
+            guest_names.append(guest_name)
```

Comment:
you can make this simpler by using make_player, i.e.
`for i in range(4):
    guest_names.append(self.make_player())
    guest_ids.append(self.player_id)
`